ansible/configs/ansible-workshops/pre_infra.yml

```diff
@@ -11,26 +12,28 @@
     - generate_env_keys
 
   tasks:
-  - debug:
-      msg: "Step 000 Pre Infrastructure"
+    - debug:
+        msg: "Step 000 Pre Infrastructure"
 
-  - name: Generate SSH keys when set_env_authorized_key
-    block:
-    - name: Generate SSH keys
-      shell: ssh-keygen -b 2048 -t rsa -f "{{ ANSIBLE_REPO_PATH }}/workdir/{{ env_authorized_key }}" -q -N ""
-      args:
-        creates: "{{ ANSIBLE_REPO_PATH }}/workdir/{{ env_authorized_key }}"
+    - name: Generate SSH keys when set_env_authorized_key
+      block:
+        - name: Generate SSH keys
+          shell: ssh-keygen -b 2048 -t rsa -f "{{ ANSIBLE_REPO_PATH }}/workdir/{{ env_authorized_key }}" -q -N ""
+          args:
+            creates: "{{ ANSIBLE_REPO_PATH }}/workdir/{{ env_authorized_key }}"
 
-    - name: fix permission
-      file:
-        path: "{{ ANSIBLE_REPO_PATH }}/workdir/{{ env_authorized_key }}"
-        mode: 0400
+        - name: fix permission
+          file:
+            path: "{{ ANSIBLE_REPO_PATH }}/workdir/{{ env_authorized_key }}"
+            mode: 0400
 
-    - name: Generate SSH pub key
-      shell: ssh-keygen -y -f "{{ ANSIBLE_REPO_PATH }}/workdir/{{ env_authorized_key }}" > "{{ ANSIBLE_REPO_PATH }}/workdir/{{ env_authorized_key }}.pub"
-      args:
-        creates: "{{ ANSIBLE_REPO_PATH }}/workdir/{{ env_authorized_key }}.pub"
-    when: set_env_authorized_key | bool
```

Comment:
It looks like this when clause got indented incorrectly. It is supposed to associate to the `block`